config/rbac/role.yaml

```diff
@@ -136,3 +136,29 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - garm-operator.mercedes-benz.com
+  resources:
+  - runners
+  verbs:
+  - create
```

Comment:
should we really allow the creation of `runners` as the controller doesn't implemented this yet

Reply:
we probably need create permissions, otherwise the operator might not be able to create runner CR with its service account... but creating runner CR manually is not much of a big deal anyway, as the controller will check if the CR has a corresponding entry in the garm DB, and if not will just clean it up.

Reply:
thanks for the explanation